components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/securityconfig/index.jsp

```diff
@@ -245,7 +247,7 @@
            <p>&nbsp;</p>
        </div>
 
-        <div id="securityConfiguration" style="<%=displayStyle%>">
+        <div id="securityConfiguration" style="<%=Encode.forCssString(displayStyle)%>">
```

Comment:
The variable displayStyle contains hardcoded values and does not contain untrusted data. Thus no need to encode specifically here